src/emulate.c

```diff
@@ -1242,7 +1311,6 @@ void rv_reset(struct riscv_t *rv, riscv_word_t pc)
 
     /* set the reset address */
     rv->PC = pc;
-    rv->insn_len = INSN_UNKNOWN;
```

Comment:
It looks like we can remove `INSN_UNKNOWN` in the enum if we are going to move `insn_len`  to `struct rv_insn_t`.

Reply:
Agree. @Risheng1128, can you check?